File: examples/python/src/03_product_details.py
import json
from client import McpSseClient, env

def main():
    sku = env("DEMO_SKU", "100080")
    c = McpSseClient(env("MCP_BASE_URL"))
    res = c.call_tool("getProduct", {"sku": sku, "imagesAsUrlOnly": True})
    print(json.dumps(res, indent=2))

if __name__ == "__main__":
    main()

File: examples/node/src/03_product_details.ts
import { createClient, getEnv } from "./_mcp_sse_client.js";

async function main() {
  const client = createClient();
  const sku = getEnv("DEMO_SKU", "100080");

  const res = await client.callTool("getProduct", { sku, imagesAsUrlOnly: true });
  console.log(JSON.stringify(res, null, 2));

  client.close();
}

main().catch((e) => {
  console.error(e);
  process.exit(1);
});
